app/views/devise/mailer/confirmation_instructions.markerb

```diff
@@ -1,4 +1,4 @@
-<%= t('devise.mailer.welcome', :email => @resource.email) %>
+<%= t('devise.mailer.welcome', :email => @resource.username) %>
```

Comment:
I feel like username is warmer than email here. I didn't change the key to avoid all translation to change.

Reply:
I think we can change the key and then change all translations via the API. I just need to find out how, but if I remember correctly, we have already done this before.

Reply:
We might be fine [just renaming the key before pushing the new files inside the WTI interface](https://webtranslateit.com/en/docs/faq#rename-a-key), but if that does not happen to affect translations, doing the same via the API is something we can do with not too much work.

@Flaburgan might changing the key here, and changing the key in `en.yml` as well? :)